packages/internet_detector.vm/tools/chocolateyuninstall.ps1

```
@@ -8,7 +8,12 @@ Disable-ScheduledTask -TaskName 'Internet Detector' -ea 0 | ForEach-Object {
     $_ | Stop-ScheduledTask
     $_ | Unregister-ScheduledTask -Confirm:$false
 }
-Stop-Process -Name "$toolName.exe" -Force -ea 0
+
+Get-Process -Name $toolName -ea 0 | ForEach-Object {
```

If the process is not running, I think this will fail.

---

@Ana06 

`-ea 0` will silently continue it.